.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
```

Comment:
The files can go in .github [according to GitHub](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) but elsewhere [they recommend otherwise](https://opensource.guide/starting-a-project/#launching-your-own-open-source-project). Happy to move.